registrations/models.py

```diff
@@ -1306,13 +1317,72 @@ class PresenceStatus:
         {"name": "contact_person"},
     )
 
+    def partially_refund_signup_group_web_store_payment(
+        self, signup_group, bypass_web_store_api_calls=False
+    ):
+        partially_refunded = False
+
+        if (
+            bypass_web_store_api_calls
+            or not (group_payment := getattr(signup_group, "payment", None))
+            or group_payment.is_fully_refunded
+        ):
+            return partially_refunded
+
+        if group_payment.status == SignUpPayment.PaymentStatus.PAID:
+            orders_data = [
+                {
+                    "orderId": group_payment.external_order_id,
+                    "items": [self.price_group.to_web_store_partial_refund_json()],
+                }
+            ]
+            resp_json = create_web_store_refunds(orders_data)
+
+            try:
+                refunded_amount = Decimal(resp_json["refunds"][0]["payment"]["total"])
+            except (LookupError, DecimalException):
+                refunded_amount = self.price_group.price
```

Comment:
The silent setting sets the amount to the partial amount that should currently be equal to what is used in Talpa. But yes, this can potentially hide issues or lead to a different value than what was used in Talpa being stored. I removed `DecimalException` from the caught exceptions and changed this to log the `LookupError` before setting refunded amount to the price group's price.